src/Http/Controllers/Admin/DashboardController.php

```diff
@@ -173,33 +178,30 @@ private function getEnabledActivities()
      * @return array
      */
     private function getAllActivities()
-    {
-        return Activity::whereIn('subject_type', $this->getEnabledActivities())
-            ->take(20)
-            ->latest()
-            ->get()
-            ->map(function ($activity) {
-                return $this->formatActivity($activity);
-            })
-            ->filter()
-            ->values();
+    {       
+        $offset = $this->config->get('twill.dashboard.offset');
+        $this->paginator = Activity::whereIn('subject_type', $this->getEnabledActivities())
+        ->latest()
+        ->paginate($offset);
+        return $this->paginator->map(function ($activity) {
+            return $this->formatActivity($activity);
+        })
+        ->filter()
+        ->values();
     }
 
     /**
      * @return array
      */
     private function getLoggedInUserActivities()
```

Comment:
Don't know how to paginate my activity, the filter is applied in frontend and not designed to use with pagination